pkg/manager/manager.go

```diff
@@ -308,7 +217,61 @@ func (m *Manager) Start() error {
 		return err
 	}
 
-	return nil
+	// Creates gRPC server and registers various services; then serves.
+	authorization := false
```

Comment:
You can define a function like this https://github.com/onosproject/onos-e2t/blob/master/pkg/manager/manager.go#L148 to start NB server and calls it from manager. We will do the same thing for controllers and SB